src/NuGet.Core/NuGet.DependencyResolver.Core/Strings.resx

```diff
@@ -120,4 +120,12 @@
   <data name="Error_PackageNotFoundWhenExpected" xml:space="preserve">
     <value>The feed '{0}' lists package '{1}' but multiple attempts to download the nupkg have failed. The feed is either invalid or required packages were removed while the current operation was in progress. Verify the package exists on the feed and try again.</value>
   </data>
+  <data name="Log_MatchingSourceFoundForPackage" xml:space="preserve">
+    <value>Package namespace matches found for package ID '{0}' are: '{1}'</value>
+    <comment>{0} - Package id {1} - list of sources</comment>
+  </data>
+  <data name="Log_NoMatchingSourceFoundForPackage" xml:space="preserve">
+    <value>Package Namespaces are configured but no matching source found for '{0}' package.</value>
```

Comment:
I already merged this one, so better to have exact same text for same issue. Also please check match case also.

https://github.com/NuGet/NuGet.Client/blob/63b8dbad9399c4eaade9a2ccdecee7056a69bc7a/src/NuGet.Core/NuGet.PackageManagement/Strings.resx#L529-L532

Reply:
fixed in the latest commit.